src/plugins/dashboard/public/application/listing/dashboard_listing.tsx

```diff
@@ -299,74 +294,50 @@ export const DashboardListing = ({
         <DashboardAppNoDataPage onDataViewCreated={() => setShowNoDataPage(false)} />
       )}
       {!showNoDataPage && (
-        <TableListView
-          createItem={!showWriteControls ? undefined : createItem}
-          deleteItems={!showWriteControls ? undefined : deleteItems}
-          initialPageSize={initialPageSize}
-          editItem={!showWriteControls ? undefined : editItem}
-          initialFilter={initialFilter ?? defaultFilter}
-          toastNotifications={core.notifications.toasts}
-          headingId="dashboardListingHeading"
-          findItems={fetchItems}
-          rowHeader="title"
-          entityNamePlural={getEntityNamePlural()}
-          tableListTitle={getTableListTitle()}
-          tableCaption={getTableCaption()}
-          entityName={getEntityName()}
-          {...{
-            emptyPrompt,
-            searchFilters,
-            listingLimit,
-            tableColumns,
-          }}
-          theme={core.theme}
-          application={core.application}
+        <TableListViewKibanaProvider
+          savedObjectTaggingApi={savedObjectsTagging!}
+          applicationStart={core.application as any}
```

Comment:
As much as I hate to add an `as any` there was no way I could make the declared interface on the component work

```
capabilities: {
  advancedSettings?: {
    save: boolean;
  };
};
```

Suggestions welcome!

Reply:
This can be mitigated by my comment above.